docs/wdc_webengine.md

```diff
@@ -0,0 +1,96 @@
+
+1. Start Tableau (with remote debugging enabled and connect to your WDC (see [Debugging a WDC in Tableau Desktop 2019.4 (and later)]({{site.baseurl}}/docs/wdc_debugging.html#debug-chrome)).  
+
+1. In the Data pane, drag out a table to the join area (if it is not there already).
+
+1. Connect the Chrome Developer Tools to your WDC. Set your breakpoints on the `getSchema` and `getData` methods, and then click **Update Now**.
+
+---
+
+### The WDC library (`wdclib`) version not supported
+
+**Details:** Starting with Tableau 2019.4, Tableau no longer supports older versions of `wdclib` (the WDC library). This includes WDC versions 1.0, 1.1, 2.0.0, and 2.0.1. For more information, see [WDC Versions]({{site.baseurl}}/docs/wdc_library_versions.html).
+
+**Resolution:** WDCs that use an old version of `wdclib` need to upgrade to the latest version. If you are a developer of a Web Data Connector, refer to the migration guide for the steps to upgrade from version 1.x, see [Upgrading from WDC Version 1.x]({{site.baseurl}}\docs\wdc_upgrade.html). If you are already using a 2.x library, you should be able to just upgrade to a newer version of the library (for example, `wdclib-2.3.latest.js`). For more information, see [WDC Versions]({{ site.baseurl }}\docs\wdc_library_versions).
+
+---
+
+### JavaScript console errors, such as `not a valid WDC` or `tableau.XXX not a valid object`
+
+**Details:** In Tableau 2019.3 and earlier, Web Data Connectors use QT WebKit as the internal browser and are initialized in a synchronous manner, which prevents the WDC code from executing until all bootstrapping code is complete. With QT WebEngine, the Chromium-based internal browser used in Tableau 2019.4 and later, it is not possible to make this guarantee. If your WDC attempts to use the `window.tableau` object before the `init` method is called, problems might occur as the API is not fully initialized.
+
+**Resolution:** For many WDCs, simply upgrading to the latest `wdclib` library will help. We have released patched versions of 2.1.x, 2.2.x, and 2.3.x to accommodate this. Make sure your WDC is using the latest version of one of those libraries: `wdclib-2.1.latest.js`, `wdclib-2.2.latest.js`, or `wdclib-2.3.latest.js`. You should also be aware that the WDC can't reliably use the `window.tableau` object until after the WDC has been initialized. For example, it is illegal to use `window.tableau.log` before the WDC has been initialized. There is no resolution to this latter issue; WDCs should simply not do this.
+
+---
+
+### Mixed-mode HTTP(S) errors
+
+**Details**: In Tableau 2019.3 and earlier, which use the QT WebKit internal browser, mixed-mode HTTP and HTTPS calls are legal. That is, a WDC is able to make an AJAX request from an HTTPS host to a HTTP host. In Tableau 2019.4 and later, the Chromium-based QT WebEngine browser reports errors when the WDC attempts mixed-mode calls.
```

Comment:
it's not just ajax requests; it's any mixed-mode. so even loading scripts or images in standard html tags will fail.

Reply:
OK. I added a bit more. 
"For example, in earlier versions of Tableau, a WDC is able to make an AJAX request from an HTTPS host to an HTTP host, or can make HTTP requests for images, scripts, or other HTML elements from an HTTPS web page. In Tableau 2019.4 and later, the Chromium-based QT WebEngine browser reports errors when the WDC attempts mixed-mode calls."